tron/config/schema.py

```diff
@@ -110,6 +110,7 @@ def config_object_factory(name, required=None, optional=None):
 ConfigMesos = config_object_factory(
     name='ConfigMesos',
     optional=[
+        'master_address',
```

Comment:
I think this should be required, especially if we're going to be initializing Mesos on start.

Reply:
hmmm...I am not sure. It makes sense for me if "master_address = None" and "enabled = False."